### ISSUE TITLE
I can't get count of children

### ISSUE THREAD

**Issue by topapp2030** [appy-one/acebase#40]
I can't get count of children
come out to me undefined

```
import { AceBase } from 'acebase'
const db = AceBase.WithIndexedDB('my_db'); // browser

function fPush(){
db.ref('game/config').push({name: 'name 1' , test : 1})
}

async function getCount(){
  const valCount = await db.ref('game/config').count()
  console.log(valCount );  // undefined
}
```

**Comment by appy-one** [comment]
It appears `count` wasn't implemented for the browser yet. I'll publish an update today

**Comment by appy-one** [comment]
Fixed and published with v1.8.3
